### PR TITLE
Fix dev script to use secret

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
Currently, if you run `npm run dev` without the secret, `sdk_token_factory` will not return a token and you won't be able to use the SDK locally

# Solution
Add secret to script

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-858-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-858-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://858-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-858-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-858-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://858-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by joninvski** [comment]
LGTM :+1:

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-858-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-858-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 1]
```diff
@@ -9,13 +9,13 @@
   },
   "license": "MIT",
   "scripts": {
-    "dev": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https",
+    "dev": "NODE_ENV=development SDK_TOKEN_FACTORY_SECRET=$SDK_TOKEN_FACTORY_SECRET webpack-dev-server --inline --hot --progress --https",
```
Comment (kopijunkie) [inline package.json]: should still work if written as just `$SDK_TOKEN_FACTORY_SECRET`

**Pull request closed without merging**